src/components/Navigation/LanguageSelector.js

```diff
@@ -102,8 +102,12 @@ LanguageSelector.defaultProps = {
 };
 
 LanguageSelector.propTypes = {
-  lang: PropTypes.string.isRequired,
+  lang: PropTypes.string,
   options: PropTypes.arrayOf(PropTypes.string)
 };
 
+LanguageSelector.defaultProps = {
```

Comment:
We already have defaultProps defined in https://github.com/TakwimuAfrica/Takwimu.AFRICA/pull/112/files#diff-c704ba2c99b0c600353009fecaa26082R97-R102. This overrides them and causes the page to crash on my end.